### PR TITLE
zaki/add_omnibus_partner

### PR DESCRIPTION
**Pull request by zakibinary**
Added omnibus partner page and it's section to partners page.

### CODE REVIEW

File: src/sass/pages/static_pages.scss [hunk 1]
```diff
@@ -286,6 +286,24 @@ body #not_authenticated_financial ul.checked > li {
...
+.omnibus-content {
+    .box {
+        min-height: 156px;
+
+        img {
+            height: 48px;
+        }
+    }
+    .steps {
+        img {
```
Comment (negarn) [inline src/sass/pages/static_pages.scss]: can we change this to `.steps img {`

File: src/sass/pages/static_pages.scss [hunk 2]
```diff
@@ -286,6 +286,24 @@ body #not_authenticated_financial ul.checked > li {
...
+            height: 43px;
+        }
+    }
+    .center-text {
```
Comment (negarn) [inline src/sass/pages/static_pages.scss]: no need to add this, we have this class in binary-style

File: src/templates/partners/omnibus.html.tt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    [% END %]
+    [% INCLUDE steps has_arrow=1 text=l('Create your own white-label application with the omnibus facility, plus all the features supported by our extensive API library. We won’t know who your clients are and vice versa') %]
+    [% INCLUDE steps has_arrow=1 text=l('Create sub-accounts to record transactions you make on behalf of your clients') %]
+    [% INCLUDE steps has_arrow=1 text=l('Customise individual access for clients who wish to trade for themselves') %]
+    [% INCLUDE steps has_arrow=1 text=l('Transfer funds between your omnibus master account and sub-accounts') %]
+    [% INCLUDE steps text=l('All deposits and withdrawals are made through your omnibus master account') %]
+</div>
+
+<div class="container">
+    <p class="center-text">[% l('Interested in our omnibus account facility? [_1].', '<a href="mailto:marketing@binary.com">Contact us now and apply as an omnibus broker</a>') %]</p>
```
Comment (negarn) [inline src/templates/partners/omnibus.html.tt]: probably we want this text `Contact us now and apply as an omnibus broker` to be translated as well, we should change this to `[% l('Interested in our omnibus account facility? [_1]Contact us now and apply as an omnibus broker[_2].', '<a href="mailto:marketing@binary.com">','</a>') %]`

Comment (kellybinary) [inline src/templates/partners/omnibus.html.tt]: i think we need to add 'Contact us now and apply as an omnibus broker' to translation

Comment (zakibinary) [reply src/templates/partners/omnibus.html.tt]: thanks, will append the changes.

**Pull request merged**